index.js

```diff
@@ -2,7 +2,15 @@
 const path = require('path');
 const Module = require('module');
 
-module.exports = (fromDir, moduleId) => {
+const _resolveFileName = (fromDir, fromFile, moduleId) => {
```

Comment:
Wondering why you prefixed this with underscores? Might have missed something in the conversation though.